.github/workflows/crates-io.yml

```
@@ -0,0 +1,29 @@
+name: Publish prusti-contracts to Crates.io
```

```suggestion
name: Publish on Crates.io
```

Long workflow names are not fully visible in https://github.com/viperproject/prusti-dev/actions.